src/main/java/TheOrangeRatchetCat.java

```diff
@@ -0,0 +1,63 @@
+import orangecat.TheOrangeCat;
+import parser.Command;
+import storage.Storage;
+import parser.Parser;
+import java.util.List;
+import java.util.ArrayList;
```

Comment:
Could import order be more consistent throughout different files? See `TheOrangeCat.java` lines 7-8.